packages/commonwealth/client/scripts/views/components/sidebar/CollapsableSidebarButtonUnscoped.tsx

```diff
@@ -0,0 +1,33 @@
+import 'components/sidebar/collapsable_button.scss';
+import React, { useState } from 'react';
+import useSidebarStore from '../../../state/ui/sidebar/index';
+import { CWIconButton } from '../component_kit/cw_icon_button';
+
+export const CollapsableSidebarButtonUnscoped = () => {
+  const { setMenu, menuName, menuVisible, setUserToggledVisibility } =
+    useSidebarStore();
+
+  const [isCollapsed, setIsCollapsed] = useState(false);
+
+  function handleToggle() {
+    const isVisible = !menuVisible;
+    setMenu({ name: menuName, isVisible });
+    setTimeout(() => {
+      setUserToggledVisibility(isVisible ? 'open' : 'closed');
+    }, 200);
+
+    setIsCollapsed(!isCollapsed);
+  }
+
+  return (
+    <div className="hover-box">
```

Comment:
our convention is that the file, component and first classname, all have the same name. please change it to CollapsableSidebarButtonUnscoped or whatever the file will be named eventually.